### ISSUE TITLE
Rename `system/ports/input`

### ISSUE THREAD

**Issue by Oldes** [Oldes/Rebol-issues#2472]
`system/ports/input` should be renamed as it is used not only for input, but also for output into standard console.
I'm just not sure, what would be the best name for it. Any idea?

**Comment by Oldes** [comment]
Related issue: https://github.com/Oldes/Rebol-issues/issues/2468

**Comment by rebolek** [comment]
I believe `/console` is the best name for the ... console.

**Comment by Oldes** [comment]
Yes... it is... but what if I would like to write (in a distant future) an interpreted console (written in Rebol instead of C), which would be based on this native one?

**Comment by rebolek** [comment]
Then you need to choose a different name for the new console I guess.
